app/src/main/res/values/strings.xml

```diff
@@ -380,6 +380,7 @@
   <!-- CompletedStoryListActivity -->
   <string name="completed_story_list_activity_title">Stories Completed</string>
   <!-- WalkthroughActivity -->
+  <string name="walk_through_activity_title">Walk Through</string>
```

Comment:
`<string name="walkthrough_activity_title">Walkthrough</string>`

@BenHenning What can be more meaningful title here?
E.g.: `App walkthrough` ?

Reply:
Done